inference/core/workflows/core_steps/sinks/roboflow/roboflow_dataset_upload.py

```diff
@@ -457,13 +457,19 @@ def register_datapoint(
     batch_name: str,
     tags: List[str],
 ) -> str:
+    inference_id = (
```

Comment:
Could u please put todo comment linking this issue: https://github.com/roboflow/inference/issues/567